README.md

```diff
@@ -544,10 +542,12 @@ https://github.com/schumilo/vUSBf
 
 https://github.com/ucsb-seclab/difuze
 
-https://github.com/compsec-snu/razzer
+https://github.com/compsec-snu/razzer (条件竞争漏洞检测)
```

Comment:
Let's keep everything in English.

Reply:
https://github.com/compsec-snu/razzer (race vulnerability detection)

Reply:
Please update the pull request with this change.